docs/tutorial/01-getting-started.md

```diff
@@ -0,0 +1,357 @@
+# Getting Started with Scaffold Stellar
+
+We'll use Scaffold Stellar to create a new Smart Contract project and walk through some simple improvements to show common blockchain interactions that will be helpful when you start crafting your own contracts. This includes:
+
+- storing data
+- authentication
+- handling transactions
+- obfuscating private data
+
+While building these features, you'll also learn the life cycle of smart contract development including
+
+- compiling
+- debugging
+- testing
+- deploying
+- and upgrading
+
+
+## ✅ Prerequisites
+
+You should have a basic understanding of using the command line and of general programming concepts. Stellar Contracts are written in a subset of Rust, although we'll walk through the code together so don't worry if this is your first time with the language.
+
+- install rust (+ cargo)
+- add rust target
+- install node (+ npm)
+- [install docker](https://docs.docker.com/desktop/) (need to run anything else?)
+- install [stellar-cli](https://github.com/stellar/stellar-cli) and [scaffold stellar](https://github.com/theahaco/scaffold-stellar) plugin
+
+TODO: rewrite our own?
+[Follow original setup instructions.](https://developers.stellar.org/docs/build/smart-contracts/getting-started/setup)
+
+
+## 🏗️ Create the Scaffold
+
+Our smart contract will be a Guess The Number game. You (the admin) can deploy the contract, randomly select a number between 1 and 10, and seed the contract with a prize. Users can make guesses and win the prize if they're correct!
+
+> ℹ️ Why a guessing game? A standard Hello World program isn't all that useful of an example. Unlike most smart contracts, there's nothing to interact with. That's why the Rust Programming Language book uses a guessing game as [their first tutorial project](https://doc.rust-lang.org/book/ch02-00-guessing-game-tutorial.html). We thought we'd do the same.
+
+Let's use the Stellar CLI tool to get a starting point. Open your terminal and navigate to the directory you keep your projects, then type:
+
+```bash
+$ stellar scaffold init my-project
```

Comment:
I _think_ we want to keep the `$` out of it, so people can copy-paste correctly. I think I've seen total newbies get confused by the dollar sign.